lib/widgets/dashboards_screen_bottom_navigation_bar.dart

```diff
@@ -5,36 +5,35 @@ import 'package:provider/provider.dart';
 class DashboardsScreenBottomNavigationBar extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    final dashboardsProvider = Provider.of<DashboardsProvider>(context);
-
-    return BottomNavigationBar(
-      backgroundColor: Theme.of(context).colorScheme.surface,
-      selectedItemColor: dashboardsProvider
-          .dashboardTabs[dashboardsProvider.dashboardTabIndex].selectedTabColor,
-      unselectedItemColor: Colors.grey,
-      currentIndex: dashboardsProvider.dashboardTabIndex,
-      type: BottomNavigationBarType.fixed,
-      items: [
-        BottomNavigationBarItem(
-          backgroundColor: Theme.of(context).colorScheme.background,
-          icon: Icon(Icons.home),
-          title: Text('Home'),
-        ),
-        BottomNavigationBarItem(
-          backgroundColor: Theme.of(context).colorScheme.background,
-          icon: Icon(Icons.star),
-          title: Text('Favourite'),
-        ),
-        BottomNavigationBarItem(
-          backgroundColor: Theme.of(context).colorScheme.background,
-          icon: Icon(Icons.block),
-          title: Text('Quarantine'),
+    return Consumer<DashboardsProvider>(
+        builder: (ctx, dashboardsProvider, child) => BottomNavigationBar(
+          backgroundColor: Theme.of(context).colorScheme.surface,
+          selectedItemColor: dashboardsProvider.dashboardTabs[dashboardsProvider.dashboardTabIndex].selectedTabColor,
+          unselectedItemColor: Colors.grey,
+          currentIndex: dashboardsProvider.dashboardTabIndex,
+          type: BottomNavigationBarType.fixed,
+          items: [
+            BottomNavigationBarItem(
+              backgroundColor: Theme.of(context).colorScheme.background,
+              icon: Icon(Icons.home),
+              title: Text('Home'),
+            ),
+            BottomNavigationBarItem(
+              backgroundColor: Theme.of(context).colorScheme.background,
+              icon: Icon(Icons.star),
+              title: Text('Favourites'),
```

Comment:
should be loaded from translations

Reply:
will be changed in change-request-part1 PR